pytest/tests/sanity/validator_switch_key_quick.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# Starts three validating nodes and one non-validating node
+# Set a new validator key that has the same account id as one of
+# the validating nodes. Stake that account with the new key
+# and make sure that the network doesn't stall even after
+# the non-validating node becomes a validator.
+
+import sys, time
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+
+from cluster import start_cluster
+
+EPOCH_LENGTH = 20
+TIMEOUT = 100
+NUM_VALIDATORS = 2
+
+client_config = {
+    "tracked_shards": [0],  # Track all shards
+    "state_sync_enabled": True,
+    "store.state_snapshot_enabled": True
```

Comment:
Why do you need those two given all nodes track all shards? 

State sync enabled on its own wouldn't work, it needs to be properly configured and a dumper node is necessary.